interp/constrextern.ml

```diff
@@ -392,7 +378,6 @@ let rec extern_cases_pattern_in_scope (scopes:local_scopes) vars pat =
 	  let args = List.map (extern_cases_pattern_in_scope scopes vars) args in
 	  let p =
 	    try
-              if !Flags.raw_print then raise Exit;
```

Comment:
I think this is a missing record_print by the way (considering it produces a CPatRecord).

Reply:
You mean in the existing code? How would you fix it?